go/chat/convsource.go

```diff
@@ -571,7 +578,7 @@ func (s *HybridConversationSource) GetMessages(ctx context.Context, convID chat1
 }
 
 func (s *HybridConversationSource) GetMessagesWithRemotes(ctx context.Context,
-	convID chat1.ConversationID, uid gregor1.UID, msgs []chat1.MessageBoxed,
+	convID chat1.ConversationID, uid gregor1.UID, msgs []chat1.MessageSummary,
```

Comment:
Yeah I think we should just nuke this function, it's not really doing anything more than `GetMessages` unless I am missing something.